### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by pritam-kr**
- Fixed some error stuff
- styling landing page
- added images for styling
- added some common styling

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
looks good to me

File: CSS/Commoncss/Nav.css [hunk 1]
```diff
@@ -9,6 +9,8 @@
     left: 0;
     right: 0;
     background-color: #fff;
+    border-bottom: 0;
```
Comment (BelideNikhil) [inline CSS/Commoncss/Nav.css:12]: remove it if you are value  to zero

File: CSS/Commoncss/Nav.css [hunk 2]
```diff
@@ -29,19 +31,21 @@
   
   .nav-logo {
     color: var(--primary-color);
-    font-size: medium;
+    font-size: 2rem;
   }
   
   .login-button {
     transition: 0.1s all;
     border-radius: 0;
```
Comment (BelideNikhil) [inline CSS/Commoncss/Nav.css:39]: same here!

File: CSS/Commoncss/Nav.css (hunk 1 above)
Comment (pritam-kr) [reply CSS/Commoncss/Nav.css:12]: ok

**Review by BelideNikhil — APPROVED** [review]
Rest all looks good!

**Pull request merged**